### PR TITLE
Measurement info panel

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium-oss/analysis-ui-components/issues/180

### CODE REVIEW

**Review by lpatiny — CHANGES_REQUESTED** [review]
I don't think the search by value is working, only key is working.

'react-inspector' in case the value is an object seems not to be implement yet.

I'm also wondering about the 2 sections, 'info' and 'meta' corresponding to the 'info' and 'meta' properties in the measurement.
There should be 2 parts (but only one query field) with a title for the 2 parts.

I just updated the https://github.com/zakodium-oss/analysis-ui-components/blob/main/public/measurements.json so that there is an info and meta property in the first measurement and one of the value is an object.

File: src/components/MeasurementInfoPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      }
+
+      return (
+        <Table.Row key={key}>
+          <ValueRenderers.Text value={key} />
+          {valueCell(value)}
+        </Table.Row>
+      );
+    });
+  const infoRows = (search: string) =>
```
Comment (lpatiny) [inline src/components/MeasurementInfoPanel.tsx:35]: Please avoid repeating the same code twice and add a function.

**Review by lpatiny — CHANGES_REQUESTED** [review]
Please first display the info and then meta.

Please also use a different background to be able to distinguish before info and meta.

File: src/components/MeasurementInfoPanel.tsx (hunk 1 above)
Comment (wadjih-bencheikh18) [reply src/components/MeasurementInfoPanel.tsx:35]: fixed thank you

**Review by lpatiny — APPROVED** [review]
Functionalities LGTM. @targos could you merge the code if it looks ok ?

**Comment by targos** [comment]
There's a conflict after I merged https://github.com/zakodium-oss/analysis-ui-components/pull/187

**Pull request merged**